client/.storybook/main.js

```diff
@@ -23,6 +32,17 @@ module.exports = {
       modules: [std_lib_path.resolve(__dirname, "../"), "node_modules/"],
     };
 
+    //Remove all existing css loaders provided by storybook
+    config.module.rules = config.module.rules.filter(
+      (f) => f.test.toString() !== "/\\.css$/"
+    );
+
+    config.module.rules.push({
+      test: /\.css$/,
+      loaders: ["style-loader", "css-loader", "postcss-loader"],
+      include: std_lib_path.resolve(__dirname, "./"),
+    });
```

Comment:
I had to add the css rules because the leaf spinner (using spin.js) would otherwise throw an error if PostCSS didn't exist

Reply:
What was the error?

Reply:
Can't remember, but I remember it had trouble parsing one of the files in the spin.js package

Reply:
Do you have some idea why that would occur in the storybook but not in the main build?

At a minimum, this needs an explanatory comment. It seems hacky that the storybook needs this loader while the client doesn't appear to use it. Even if you can't find an alternate solution, please try and understand (and document) more of the underlying problem here if possible.

I had a third (and maybe fourth) point I was going to make, but while thinking about it I may have noticed what was actually wrong my self. I'm going to play around with this a little locally and get back to you.

Reply:
👍 Will leave a comment

I remember reading something about storybook having its own css loader and won't use anything else unless specified or something along those lines. In which case the client is probably adding postcss loader on its own when plugging in spin.js.

Reply:
See #985, fixed there.

Reply:
Webpack's pretty arcane, and this was extra obtuse because I'd been slamming together our own rules with their defaults, and hadn't really thought too much about that because it was all working initially. The error was fairly misleading too, especially because we had a vestigial import for PostCSS in our package.json, from however long ago we used that. So, this definitely wasn't an easy one.

Still, and this is relevant to the use of debounce elsewhere in this PR, if you don't know _why_ a fix works, or what was actually broken in the first place, then it's really just a hack (no matter how clean or simple the "fix" itself might seem). Sometimes an error message is from a symptom, not from the actual disease, and treating the symptoms won't actually get us anywhere longterm. Sometimes you just need to go do a lot of googling/dig around in the actual source code of the library you're having problems with/read about how the JS event loop works 🤷 .

Or, again, at the very least slap a big comment on the "fix" to make clear that it's just a monkey patch, and that the real problem needs more investigation (comments are basically hack alerts, ha). Luckily my PR there should sort this one out.

Case in point, although it was a lot more subtle, we wouldn't have run in to this now if I'd bothered to roll up my sleeves and dig in to storybook's GitHub to find their default storybook config in the first place, ha. I must have had to much confidence in my own understanding of what my fix was accomplishing back when I fully merged their rules with ours.